streaming/python/config.py

```diff
@@ -26,3 +25,38 @@ class Config:
     FLOW_CONTROL_TYPE = "streaming.flow_control_type"
     WRITER_CONSUMED_STEP = "streaming.writer.consumed_step"
     READER_CONSUMED_STEP = "streaming.reader.consumed_step"
+
+    # state backend
+    CP_STATE_BACKEND_TYPE = "streaming.context-backend.type"
+    CP_STATE_BACKEND_MEMORY = "memory"
+    CP_STATE_BACKEND_LOCAL_FILE = "local_file"
+    CP_STATE_BACKEND_DEFAULT = CP_STATE_BACKEND_MEMORY
+
+    # local disk
+    FILE_STATE_ROOT_PATH = "streaming.context-backend.file-state.root"
+    FILE_STATE_ROOT_PATH_DEFAULT = "/tmp/ray_streaming_state"
+
+    # checkpoint
+    JOB_WORKER_CONTEXT_KEY = "jobworker_context_"
```

Comment:
Should not be end with `_`.

Reply:
This is a prefix key, for example `jobworker_context_0_1` I think it's OK.